ptsprojects/zephyr/post_tc.py

```diff
@@ -23,7 +23,7 @@
 
 CONFIG_PROC = None
 # XXX: Fill me - nrfjprog path example: /home/user/tool/nrfjprog
-CONFIG_PATH = None
+CONFIG_PATH = '/usr/bin/nrfjprog'
```

Comment:
Is this needed? This path may be specific to your machine. Do we use it in a standard auto-pts run?